### PR TITLE
[BUGFIX] : NFTs do not load in the gallery when displayed in list view

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### 📝 Description

NFTs do not load in the gallery when displayed in list view
### ❓ Context

- **Impacted projects**: `LLD` 
- **Linked resource(s)**: [LIVE-8374] 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 📸 Demo

https://github.com/LedgerHQ/ledger-live/assets/112866305/82ca6446-53c6-42ac-8a08-83dc0b85813a

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-8374]: https://ledgerhq.atlassian.net/browse/LIVE-8374?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/screens/nft/useOnScreen.ts
Comment (mcayuelas-ledger) [inline apps/ledger-live-desktop/src/renderer/screens/nft/useOnScreen.ts:1]: Maybe this one could be moved somewhere more generic?
Any ideas?

File: .changeset/popular-drinks-repeat.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": patch
+---
+
+NFTs do not load in the gallery when displayed in list view
```
Comment (sshmaxime) [inline .changeset/popular-drinks-repeat.md:5]: I think we should be more explicit when writing changeset, like: `Fixing not loading NFTs in the gallery when displayed in list view`

File: apps/ledger-live-desktop/src/renderer/screens/nft/Gallery/Gallery.tsx [hunk 2]
```diff
@@ -43,6 +44,11 @@ const ClickableCollectionName = styled(Box)`
     cursor: pointer;
   }
 `;
+
+const Footer = styled.footer`
```
Comment (sshmaxime) [inline apps/ledger-live-desktop/src/renderer/screens/nft/Gallery/Gallery.tsx:48]: There are 2 of this component, maybe we could unify them ?

File: apps/ledger-live-desktop/src/renderer/screens/nft/useOnScreen.ts
Comment (sshmaxime) [reply apps/ledger-live-desktop/src/renderer/screens/nft/useOnScreen.ts:1]: This should indeed be moved somewhere else. We are lacking for a place to store useful hooks, that's really not ideal. We should have an architecture meeting or something like that, as suggested @KVNLS. cc @cgrellard-ledger

**Review by sshmaxime — APPROVED** [review]

**Review by cgrellard-ledger — APPROVED** [review]

**Comment by cgrellard-ledger** [comment]
LGTM !

**Pull request merged**